handlers/reload.yml

```diff
@@ -0,0 +1,41 @@
+# file: postgresql/handlers/reload.yml
+  
+#
+# Reload PostgreSQL
+#
+- name: PostgreSQL | <= v9.4, must restart to implement parameter changes
```

Comment:
The `- name` is ignored, because I have to use `include_tasks` instead of `import_tasks`.  It's not the prettiest output, but it's the only way I've found to make it actually work.